source/_posts/2022-04-16-device-to-run-home-assistant.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Buying a device to run Home Assistant (April 2022 edition)"
+description: "Chip shortages and global supply chain issues make getting a device complicated. Here is our latest overview."
+date: 2022-04-16 00:00:00
+date_formatted: "April 17, 2022"
+author: Paulus Schoutsen
+author_twitter: balloob
+comments: true
+categories:
+- Announcements
+---
+
+Home Assistant is a home automation platform with a focus on privacy and local control. It can function fully without an internet connection. Home Assistant is easy in day-to-day use, and we've been spending a lot of time on streamlining this experience.
```

Comment:
```suggestion
Home Assistant is a home automation platform with a focus on privacy and local control. It can function fully without an internet connection. Home Assistant is easy in day-to-day use, and we've been spending a lot of time streamlining this experience.
```